arcgis-ios-sdk-samples/Geometry/Project/README.md

```diff
@@ -0,0 +1,22 @@
+#Project
+Project a point to another spatial reference.
```

Comment:
A couple of formatting considerations applicable to all headings here -- 
1) try adding a space between `#` and `Project` to ensure this becomes a heading
2) add a new line between the heading and the text below to ensure these don't get conflated to a single line